python-squid/src/main/java/org/sonar/python/metrics/FileLinesVisitor.java

```diff
@@ -45,37 +48,94 @@
 
   private final FileLinesContextFactory fileLinesContextFactory;
 
+  private boolean seenFirstToken;
+
+  private final boolean enableNoSonar;
+
+  private final boolean ignoreHeaderComments;
+
+  private Set<Integer> noSonar = Sets.newHashSet();
   private Set<Integer> linesOfCode = Sets.newHashSet();
   private Set<Integer> linesOfComments = Sets.newHashSet();
+  private Set<Integer> linesOfDocstring = Sets.newHashSet();
   private final FileSystem fileSystem;
   private final Map<InputFile, Set<Integer>> allLinesOfCode;
 
-  public FileLinesVisitor(FileLinesContextFactory fileLinesContextFactory, FileSystem fileSystem, Map<InputFile, Set<Integer>> linesOfCode) {
+  public FileLinesVisitor(
+      FileLinesContextFactory fileLinesContextFactory,
+      FileSystem fileSystem,
+      Map<InputFile, Set<Integer>> linesOfCode,
+      boolean enableNoSonar,
```

Comment:
@pynicolas You're right. Removed. Now equivalent to enableNoSonar always set to true.